CHANGELOG_UNRELEASED.md

```diff
@@ -26,6 +26,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - in `poly.v`
   + lemmas `coef_prod_XsubC`, `coefPn_prod_XsubC`, `coef0_prod_XsubC`
 
+- in `ssralg.v`
+  + definitions `bool_zmodType`, `bool_comRingMixin`, `bool_ringType`,
+    `bool_comRingType`, `bool_unitRingMixin`, `bool_unitRingType`,
+    `bool_comUnitRingType`, `bool_idomainMixin`, `bool_fieldMixin`,
+    `bool_idomainMixin`, `bool_idomainType`, `bool_fieldType`
+  + lemmas `mulVb`, `invb_out`
+
+- in `finalg.v`
+  + definitions `bool_finZmodType`, `bool_finRingType`, `bool_finComRingType`,
+    `bool_finUnitRingType`, `bool_finComUnitRingType`, `bool_finIdomainType`,
+    `bool_finFieldType`, `bool_decFieldType` 
```

Comment:
Actually this changelog is irrelevant, specific names of canonical structures should not be mentioned (only their existence) and private lemmas (justifying structure instanciation) should be omitted.